mlflow/store/db/utils.py

```diff
@@ -153,6 +153,10 @@ def _upgrade_db(engine):
         command.upgrade(config, "heads")
 
 
+def _upgrade_url(url):
+    _upgrade_db(create_sqlalchemy_engine_with_retry(url))
```

Comment:
In next commits the function was removed. But how does it address incompatible db issue if without upgrading db ?

Reply:
Shall we use file store in these mlflow 1.28 tests instead ?

Reply:
@WeichenXu123 You're commenting to an old commit. I already revert this change.

Reply:
> But how does it address incompatible db issue if without upgrading db ?

We can use a file store.